modules/phase_field/src/auxkernels/TestEBSDAux.C

```diff
@@ -24,6 +24,11 @@ TestEBSDAux::TestEBSDAux(const InputParameters & parameters) :
 {
 }
 
+TestEBSDAux::~TestEBSDAux()
+{
+  delete _val;
```

Comment:
Hmm, I don't remember why I new anything there. On the phone now but I'll look later if we can change that to references.